tests/hedgehog_test_runner.py

```diff
@@ -16,29 +16,50 @@ def read_yaml(path):
             print(exc)
 
 
-def read_metadata(testbed_data):
-    metadata = False
+def run_cmd_on_dut(testbed_data, cmd):
```

Comment:
Why cannot we use `<duthost>.command(...)` method?

Reply:
`duthost` is initiated after this script. 
I didn't want to init it inside this script, 
I just want to make it less complicated and more independent from other community code